troubleshooting/faq.md

```diff
@@ -6,6 +6,48 @@ sidebar: sidebars/troubleshooting.html
 
 # Frequently Asked Questions
 
+### Can I use pulp-admin with katello?
+
+While it is typically not encouraged, you can use pulp-admin with katello for troubleshooting purposes.
```

Comment:
Reads hard for me, a suggested re-wording with reasoning: "We do not encourage the use of pulp-admin because it has the potential to get data out of sync. However, pulp-admin can be useful when troubleshooting Katello."